src/objclass/class_api.cc

```diff
@@ -636,7 +636,8 @@ int cls_cxx_list_watchers(cls_method_context_t hctx,
 
 int cls_gen_random_bytes(char *buf, int size)
 {
-  return get_random_bytes(buf, size);
+  get_random_bytes(buf, size);
+  return 0;
```

Comment:
Surely this function should just return void?

Reply:
i'm assuming this is part of the public cls api, so i opted not to change it